.drone.yml

```
@@ -16,10 +16,11 @@ steps:
   - name: test
     image: node:18
     commands:
-      - npm ci
-      - npm run build
-      - npm run lint
-      - npm run test
+      # - npm ci
+      # - npm run build
+      # - npm run lint
+      # - npm run test
+      - echo "Skipping for testing"
```

TODO: revert before merge 

```suggestion
      - npm ci
      - npm run build
      - npm run lint
      - npm run test
```